<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelWidget</class>
 <widget class="QWidget" name="LevelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>75</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>75</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Pristina</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Choose the level of game</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hardButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pristina</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>hard</string>
   </property>
  </widget>
  <widget class="QPushButton" name="easyButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>119</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pristina</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>easy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="veryEasyButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pristina</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>very easy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mediumButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pristina</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>medium</string>
   </property>
  </widget>
  <widget class="QPushButton" name="veryHardButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pristina</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>very hard</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
